SU-SWS/stanford_story_page
# HSDO-970 Removed page template in favor of css and context

# READY FOR REVIEW

# Summary
- Removed the page template and use context to disable the sidebar regions and css to make full width.

# Needed By (Date)
- 8/17

# Urgency
- low

# Steps to Test

1.Checkout branch and branch in [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/69)
2. clear cache
3. create story page
4. validate its still full width and looks very similar to what it was before the template removal.
5. Might check SOE if its using this.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/69

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
